liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/StartDebugMojoSupport.java

```diff
@@ -218,13 +346,45 @@ private void writeBootstrapProperties(File file, Map<String, String> properties)
         }
     }
 
-    private void writeJvmOptions(File file, List<String> options) throws IOException {
+    private void writeServerEnvProperties(File file, Map<String, String> mavenProperties) throws IOException {
         makeParentDirectory(file);
         PrintWriter writer = null;
         try {
             writer = new PrintWriter(file, "UTF-8");
             writer.println(HEADER);
-            for (String option : options) {
+            for (Map.Entry<String, String> entry : mavenProperties.entrySet()) {
+                String key = entry.getKey();
+                writer.print(key);
+                writer.print("=");
+                String value = entry.getValue();
+                writer.println((value != null) ? value.replace("\\", "/") : "");
```

Comment:
Why does a replace need to be done here?

Reply:
I simply copied what the other methods here were doing.